org.eclipse.xtext.xtext.ui/src/org/eclipse/xtext/xtext/ui/XtextHyperlinkHelper.java

```diff
@@ -76,7 +77,7 @@ protected void createLinksToBase(ITextRegion nameLocation,	AbstractRule rule, IH
 				URI uri = uriConverter.normalize(EcoreUtil.getURI(overwritten));
 
 				XtextHyperlink result = getHyperlinkProvider().get();
-				result.setHyperlinkRegion(toRegion(nameLocation));
+				result.setHyperlinkRegion((IRegion)toRegion(nameLocation));
```

Comment:
change intended?

Reply:
Yes. I wanted to get rid of the deprecation warning.

Reply:
i have the feeling we kept it the old way back then for a reason.

Reply:
https://github.com/eclipse/xtext-eclipse/pull/1146

Reply:
I changed it in a different way to get rid of the warning.